decidim-assemblies/app/models/decidim/assembly.rb

```diff
@@ -47,5 +56,17 @@ def hashtag
     def to_param
       slug
     end
+
+    def private_space?
+      private_space
+    end
+
+    def self.private_space
+      where(private_space: true)
+    end
+
+    def self.public_assembly
```

Comment:
`def public_assemblies`, maybe? Although this might clash in meaning with `Decidim::Participable.public_spaces`...

Maybe something like `def non_private_assemblies`?